src/couch_replicator_connection.erl

```diff
@@ -137,10 +137,15 @@ handle_info({'EXIT', Pid, Reason}, State) ->
             ok;
         [Worker] ->
             #connection{host=Host, port=Port} = Worker,
-            couch_log:info(
-                "Replication connection to: ~p:~p died with reason ~p",
-                [Host, Port, Reason]
-            ),
+            case Reason =/= normal of
```

Comment:
pull this maybe logic into its own function so we don't distract from the main purpose of this function?